app/services/edge_cache/bust.rb

```diff
@@ -43,73 +41,17 @@ def call
     def determine_provider
       if fastly_enabled?
         "fastly"
-      elsif nginx_enabled? && nginx_available?
```

Comment:
We can remove `nginx_available?` out of here now since it can just live in the `EdgeCache::Bust::Nginx` class instead.